CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 ## Changelog
 
+### 2.88
+
+Release date: 2021-02-16
+
+* Internal: Update various dependencies ([PR #404](https://github.com/jenkinsci/workflow-cps-plugin/pull/404), [PR #405](https://github.com/jenkinsci/workflow-cps-plugin/pull/405), [PR #412](https://github.com/jenkinsci/workflow-cps-plugin/pull/412), [PR #413](https://github.com/jenkinsci/workflow-cps-plugin/pull/413))
```

Comment:
nit: I like to list the most important changes like fixes or new features first and the internal stuff last.